internal/pkg/deploy/cloudformation/stack/scheduled_job.go

```diff
@@ -0,0 +1,299 @@
+	if err != nil {
+		return "", fmt.Errorf("convert the sidecar configuration for job %s: %w", j.name, err)
+	}
+
+	schedule, err := j.awsSchedule()
+	if err != nil {
+		return "", fmt.Errorf("convert schedule for job %s: %w", j.name, err)
+	}
+
+	stateMachine, err := j.stateMachine()
+	if err != nil {
+		return "", fmt.Errorf("convert retry/timeout config for job %s: %w", j.name, err)
+	}
+
+	content, err := j.parser.ParseScheduledJob(template.WorkloadOpts{
+		Variables:          j.manifest.Variables,
+		Secrets:            j.manifest.Secrets,
+		NestedStack:        outputs,
+		Sidecars:           sidecars,
+		ScheduleExpression: schedule,
+		StateMachine:       stateMachine,
+		LogConfig:          j.manifest.LogConfigOpts(),
+	})
+	if err != nil {
+		return "", fmt.Errorf("parse scheduled job template: %w", err)
+	}
+	return content.String(), nil
+}
+
+// awsSchedule converts the Schedule string to the format required by Cloudwatch Events
+// https://docs.aws.amazon.com/lambda/latest/dg/services-cloudwatchevents-expressions.html
+// Cron expressions must have an sixth "year" field, and must contain at least one ? (either-or)
+// in either day-of-month or day-of-week.
+// Day-of-week expressions are zero-indexed in Golang but one-indexed in AWS.
+// @every cron definition strings are converted to rates.
+// All others become cron expressions.
+func (j *ScheduledJob) awsSchedule() (string, error) {
+	if j.manifest.Schedule == "" {
+		return "", fmt.Errorf("missing required field schedule in manifest for job %s", j.name)
+	}
+
+	// Try parsing the string as a cron expression to validate it.
+	if _, err := cron.ParseStandard(j.manifest.Schedule); err != nil {
+		return "", fmt.Errorf("schedule %s for job %s is not a valid cron expression or definition string", j.manifest.Schedule, j.name)
+	}
+	var scheduleExpression string
+	var err error
+	switch {
+	case strings.HasPrefix(j.manifest.Schedule, every):
+		scheduleExpression, err = toRate(j.manifest.Schedule[len(every):])
+		if err != nil {
+			return "", fmt.Errorf("parse fixed interval: %w", err)
+		}
+	case strings.HasPrefix(j.manifest.Schedule, "@"):
+		scheduleExpression, err = toFixedSchedule(j.manifest.Schedule)
+		if err != nil {
+			return "", fmt.Errorf("parse preset schedule: %w", err)
+		}
+	default:
+		scheduleExpression, err = toAWSCron(j.manifest.Schedule)
+		if err != nil {
+			return "", fmt.Errorf("parse cron schedule: %w", err)
+		}
+	}
+	return scheduleExpression, nil
+}
+
+func toRate(durationString string) (string, error) {
+	d, err := time.ParseDuration(durationString)
+	if err != nil {
+		return "", fmt.Errorf("invalid duration %s", durationString)
+	}
+	// Check that rates are not specified in units smaller than minutes
+	floatMinutes := d.Minutes()
+	if floatMinutes != math.Round(floatMinutes) {
+		return "", fmt.Errorf("duration must be a whole number of minutes or hours")
+	}
+
+	if d < time.Minute*1 {
+		return "", errors.New("duration must be >= 1 minute")
```

Comment:
What do you think of this?
```suggestion
		return "", errors.New("duration must be greater than or equal to 1 minute")
```